statsmodels/tools/rng_qrng.py

```diff
@@ -1,9 +1,21 @@
 import numpy as np
 import scipy.stats as stats
 
+_future_warn = """\
+Passing `None` as the seed currently return the NumPy singleton RandomState
+(np.random.mtrand._rand). After release 0.13 this will change to using the
+default generator provided by NumPy (np.random.default_rng()). If you need
+reproducible draws, you should pass a seeded np.random.Generator, e.g.,
+
+import numpy as np
+seed = 32839283923801
```

Comment:
Maybe it could help to explain that a good quality seed can be generated with `np.random.SeedSequence().entropy`